src/components/EditorSideBar/EditorSideBarContent.tsx

```diff
@@ -21,6 +23,7 @@ const showContainerItem = (iconClickTitle: string) => {
 };
 
 const SideBarContainer = (props: Props) => {
+  const { data: courtsData } = useGetCourtsQuery(0);
```

Comment:
What does parameter 0 do?

Reply:
Actually, it is a syntax altert from typescript in my view. It is expected 1-2 arguments. As we are gonna query all courts data, we could put 0 or null here to avoid error
<img width="992" alt="Screen Shot 2022-10-17 at 11 38 49 am" src="https://user-images.githubusercontent.com/85828920/196084522-5dcffaf8-47fd-4504-90e8-79453111af76.png">

Reply:
> Actually, it is a syntax altert from typescript in my view. It is expected 1-2 arguments. As we are gonna query all courts data, we could put 0 or null here to avoid error <img alt="Screen Shot 2022-10-17 at 11 38 49 am" width="992" src="https://user-images.githubusercontent.com/85828920/196084522-5dcffaf8-47fd-4504-90e8-79453111af76.png">

For this type issue, try to use generic to limit the arg and res of the getQuery functions like:
getData: builder.query<responseDataType, argDataType>({}), here the argDataType should be void. If you do not check the type, the default will be <any, any>.